inc/template-functions.php

```diff
@@ -44,3 +44,32 @@ function ampconf_excerpt_more( $more_string ) {
 	return $more_string;
 }
 add_filter( 'excerpt_more', 'ampconf_excerpt_more' );
+
+/**
+ * Filters the archive title and wraps the type of archive in a span element.
+ *
+ * @param string $title Archive title to be displayed.
+ *
+ * @return string $title
+ */
+function ampconf_get_the_archive_title( $title ) {
+	$parts = explode( ':', $title );
```

Comment:
Some languages may translate “:” differently for the strings found in `get_the_archive_title()` so this won't always work.